gitlabform/gitlab/pipelines.py

```diff
@@ -0,0 +1,14 @@
+from gitlabform.gitlab.core import GitLabCore
+
+
+class GitLabPipelines(GitLabCore):
+
+    def get_pipelines(self, project_and_group_name, branch):
+        pipelines = self._make_requests_to_api("projects/%s/pipelines?ref=%s", (project_and_group_name, branch))
```

Comment:
Please check if this endpoint is not paginated. In that case you need to do pass appropriate parameter to `_make_requests_to_api` method, see https://github.com/egnyte/gitlabform/commit/ee00182f973ffbf32766c23a6000548c3e2b999f as an example.

Reply:
Ok will do. But is it necessary if we are taking last pipeline?